apps/autoscheduler/FunctionDAG.cpp

```diff
@@ -996,66 +996,83 @@ void FunctionDAG::featurize() {
 
             Featurizer featurizer(node.func, stage);
 
-            Definition def = node.func.definition();
-            if (stage_idx > 0) {
-                def = node.func.updates()[stage_idx - 1];
-            }
-            stage.features = PipelineFeatures();
-
-            for (auto v : def.values()) {
-                featurizer.visit_store_args(node.func.name(), v.type(), def.args());
-                v = common_subexpression_elimination(simplify(v));  // Get things into canonical form
-                v.accept(&featurizer);
-            }
-            for (auto v : def.args()) {
-                v = common_subexpression_elimination(simplify(v));  // Get things into canonical form
+            if (node.func.extern_definition_proxy_expr().get()) {
+                // Extern function call with a proxy implementation specified: generate the featurization from the proxy
+                Expr v = common_subexpression_elimination(simplify(node.func.extern_definition_proxy_expr()));  // Get things into canonical form
```

Comment:
Nit: this line seems awfully long.

Reply:
For some reason clang-format won't improve the formatting. I manually split the code.